lib/src/main/java/com/auth0/android/lock/internal/configuration/Configuration.java

```diff
@@ -273,9 +273,9 @@ public int getPasswordlessMode() {
         return passwordlessMode;
     }
 
-    @PasswordStrength
-    public int getPasswordPolicy() {
```

Comment:
Change to a public API .... BC?

Reply:
classes under the `internal` package can change. There's a header on (some of) the files explaining that.